lib/xapi/problem.rb

```diff
@@ -63,7 +67,12 @@ def md
       path('%s.md' % slug)
     end
 
-    def path(f)
+    def path(file_name)
+      old_path = root_path(file_name)
+      File.exist?(old_path) ? old_path : File.join(meta_dir, file_name)
+    end
+
+    def root_path(f)
```

Comment:
Probably an artifact from me: I've mostly been using 'f' for file names whenever I talk about paths, because this is an obvious mapping to me. That's probably a bad thing, since this is Ruby, and the convention is to use the full name for things.